### ISSUE TITLE
Dropdown : Incorrect role is defined for control under the Dropdown pane in code.

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10906]
**User Experience:**
Screen reader user will be confused if incorrect role is defined for the controls, 

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 19005.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Edge Browser Version:
Microsoft Edge: 44.19005.1.0
Microsoft EdgeHTML: 18.19005
Chrome Browser Version:
Chrome: 77.0.3865.120

**Usages:**
All

Repro Steps:
1.https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown in Edge and Turn on Narrator (Ctrl+Win+enter).
2.Navigate to 'Basic input' button and activate it.
3.Navigate to 'Dropdown' link and activate it.
4.'Dropdown' pane will be appeared.
5.Navigate to any dropdown control and verify the issue.

**Actual Result:**
Screen reader conveying incorrect role of the control. Screen reader announces 'Basic uncontrolled example button collapsed has pop up'.

**Expected Result:**
Screen reader should announce the correct announcement of the control. Screen reader should announce 'Basic uncontrolled example Combo box'.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment:**
![image](https://user-images.githubusercontent.com/55537167/67094639-d9708680-f1d1-11e9-9767-872b5c31bbc3.png)
[OneDrive_2019-10-18.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3743926/OneDrive_2019-10-18.zip)

**Comment by marygans** [comment]
Hi @999-Dev thank you for submitting this issue!
I am unable to repro this using Narrator + Edge. When I focus on the dropdown this is what is announced: "Basic uncontrolled example. Apple Selected. Selection contains 7 items. "
When I expand the dropdown this is what is announced: "Basic uncontrolled example. Apple, 1 of 7 Selected."
I will share a recording for what I am experiencing momentarily 

Thanks!

**Comment by marygans** [comment]
Below is the the Narrator output captured from Narrator Buddy. As you can see, the output is different from what you have expressed and the actual behavior is expected. 

Output: 

Basic uncontrolled example, Apple selected, selection contains <say-as interpret-as="spell">7</say-as> items, 
Basic uncontrolled example, Apple, <say-as interpret-as="spell">1</say-as> of <say-as interpret-as="spell">7</say-as>, selected, 
Basic uncontrolled example, Apple selected, selection contains <say-as interpret-as="spell">7</say-as> items, 

<img width="677" alt="narratorbuddy-captured" src="https://user-images.githubusercontent.com/38991459/67133054-f9974a00-f1bf-11e9-8be3-a0d995af1acc.png">

**Comment by 999-Dev** [comment]
Hi, @marygans 
Can you please refer the attachment which I have shared with you that what narrator announces when focus moves to the control. Still narrator announces the same.

**Comment by marygans** [comment]
Hi @999-Dev! this doesn't seem to be an issue anymore, please confirm when you get a chance. 
Thanks!!
